.github/workflows/check.yml

```diff
@@ -25,6 +25,12 @@ jobs:
       - x64
       - sre
     steps:
+      - name: Clean up
+        continue-on-error: true
+        run: |
+          sudo chown -R $USER:$USER $GITHUB_WORKSPACE
+          docker system prune --force --all --volumes
```

Comment:
Won't this remove our caching?

Reply:
No it won't, cached files are saved to a specified path